modules/post/multi/gather/jenkins_gather.rb

```diff
@@ -33,6 +33,7 @@ def initialize(info = {})
     )
     register_options(
       [
+        OptString.new('JENKINS_HOME', [ false, 'Set to the home directory of Jenkins. The Linux versions default to /var/lib/jenkins, but C:\\ProgramData\\Jenkins\\.jenkins on Windows.', ]),
```

Comment:
```suggestion
        OptString.new('JENKINS_HOME', [ false, 'Set to the home directory of Jenkins. The Linux versions default to /var/lib/jenkins, but C:\\\\ProgramData\\\\Jenkins\\\\.jenkins on Windows.', ]),
```

On console the description outputs the windows path as `C:\ProgramData\Jenkins\.jenkins`. Which when a user attempts to paste it into the options it will again be escaped with results in `jenkins_home => C:ProgramDataJenkins.jenkins`. So we could update that description to avoid copy pasta issues for users.

Current output:
![image](https://github.com/rapid7/metasploit-framework/assets/69522014/fdeb058c-617c-42ff-8d96-0829f61c1973)